docs/completeExamples/completeExamples.md

```diff
@@ -0,0 +1,44 @@
+# Complete Examples
+A number of examples of complete events are provided below, in order to illustrate use of the schema.
+
+N.B. All entities including organisations, persons, devices and applications are imaginary.
+All details (e.g. MAC Address, Name, IP Address) are similarly imaginary and any relationship to any 
+real entities is entirely coincidental!
+
+In some places ```<UserDetails>``` have been abbreviated for clarity using an elipsis (...).
+
+In a real-world situation, a design decision would be needed - when to leave the event more concise using
+only an ID and when to join with additional information.  Data such as user details
+change slowly, but they _do change_, e.g. as people get promoted, change role, etc.  Therefore, such a joining 
+process would utilise the values for these additional data fields that were correct at the time the event
+was generated.  Certain tools such as [Stroom](https://github.com/gchq/stroom-docs/blob/master/README.md 
+"Stroom on Github") support this, and are sufficiently flexible to allow this enrichment process to take place
+during initial normalisation prior to storage, during analysis or at query time / inspection.
```

Comment:
It may be worth noting here that an enrichment at query/analysis time would require a versioned reference data source, i.e. to be able to get the enriched data as it looked at time X.